client/js/components/statement/assessmentTable/DpTable.vue

```diff
@@ -170,11 +170,12 @@
       <!-- Loop statements in default viewMode -->
       <template
         v-else
-        v-for="statement in statements">
+        v-for="(statement, key, index) in statements">
         <dp-assessment-table-card
           :csrf-token="csrfToken"
+          :data-cy="`statementCard:index:${index}`"
           :ref="'itemdisplay_' + statement.id"
-          :key="`statement:${statement.id}`"
+          :key="`statement:${key}`"
```

Comment:
why this change?

Reply:
When I added only index to v-for like `v-for="(statement, index) in statements"`
then data-cy resault: `data-cy="statementCard:index:fa0218f0-0175-4ead-be0f-e3dbffc6e27d"`

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3163/commits/8550398eebb5cc449d751c28bf462ff25a872dfe